test/coordinator-test.js

```diff
@@ -0,0 +1,29 @@
+import { assert } from 'chai';
+import os from 'os';
+import sinon from 'sinon';
+import { getDefaultCPUs, getWorkerCount } from '../lib/coordinator';
+
+describe('coordinator', () => {
+  it('default method returns correct number of cpus', () => {
+    assert.equal(getDefaultCPUs(5), 4, 'getDefaultCPUs returns n - 1 CPUs');
+    assert.equal(getDefaultCPUs(), 1, 'getDefaultCPUs falls back to 1 CPU');
```

Comment:
Since this is exported, let's add a test, just in case, for what happens when 0 or a negative number is passed in (I'd expect it to throw or return `NaN`)